metadata/en-US/full_description.txt

```diff
@@ -1,3 +1,7 @@
 Tailscale is a mesh VPN alternative that makes it easy to connect your devices, wherever they are. No more fighting configuration or firewall ports. Built on WireGuard®, Tailscale enables an incremental shift to zero-trust networking by implementing “always-on” remote access. This guarantees a consistent, portable, and secure experience independent of physical location.
 
 WireGuard is a registered trademark of Jason A. Donenfeld.
+
+Anti-Features:
+* NonFreeNet - uses a non-free network service which is impossible, or not easy to replace without rebuilding the app
```

Comment:
```suggestion
* NonFreeNet - Tailscale provides a <a href="https://tailscale.com/pricing/">Free plan for up to 3 users</a>. <a href="https://github.com/juanfont/headscale/blob/main/docs/android-client.md">Headscale</a> provides a self-hosted alternative.
```

Reply:
Not sure I follow, this is about Tailscale, whatever else exists out there is not related here.

Reply:
The client can be switched from using the Tailscale hosted control plane to a self-hosted control plane using Headscale. The link points to the instructions on how to switch the app from using the hosted control plane to a self-hosted instance.

Reply:
Right, did not open the link on mobile :+1: